### ISSUE TITLE
Auto scroll down for new messages doesn't work

### ISSUE THREAD

**Issue by jnpf** [vector-im/riot-web#11805]
If a new message drops in, the window doesn't scroll down automatically? 
<img width="188" alt="grafik" src="https://user-images.githubusercontent.com/44313602/72059376-2d22d580-32d2-11ea-96be-fa5d9b419a56.png">

### Version information

- **Platform**: web (https://riot.im/app/ in Mac-App "Rambox")

For the web app:

- **Browser**: Rambox (Free Version)
- **OS**: macOS
- **URL**: https://riot.im/app/

**Comment by turt2live** [comment]
have you scrolled up at all?

**Comment by jnpf** [comment]
You mean to unread messages in the past?
There are no marks for still unread messages in this room.

**Comment by turt2live** [comment]
No, I mean using the scroll wheel to look at previous messages.

**Comment by jnpf** [comment]
Oh no i didn't, thats the problem.

**Comment by alturiak** [comment]
I'm seeing this as well on Riot for Windows (electron).
It seems to happen only in windowed mode, not in fullscreen.
Upon changing rooms, the "Scroll down to bottom"-marker appears. Scrolling down sometimes makes it disappear, but the room won't scroll down on new messages (neither my own nor from others).
I've only recently started using riot in windowed mode, so I wouldn't know if this behavior started appearing with any specific version.

**Comment by mmueller** [comment]
I've started seeing this on one riot-desktop install (and only one, out of several different machines). It's a recent install of Ubuntu 19.10 with riot 1.6.1 installed via the usual apt method.

In some rooms, the jump-to-bottom button never goes away, even if I click it or scroll all the way to the bottom. And the room won't automatically scroll with new messages, as if I've scrolled up a bit, even though I haven't.

Actually... just playing with it so I could describe the behavior, it seems like if I scroll about 1 pixel backward, the button briefly disappears. I tried resizing my riot-desktop window to be slightly smaller (vertically) and now all of the rooms are scrolling correctly and hiding the jump-to-bottom button when I'm caught up! There must be an off-by-one error or something here.

**Comment by MV-SH** [comment]
Just adding a bit of "me too" as this issue is still present as of today (riot-desktop current nightly running on Arch Linux). It was driving me crazy for the past few days as I couldn't find the real cause for it, but looking up the comments from @alturiak and @mmueller pointed me in the right direction.

As changing the main window size is not an option for my desktop setup (tiled windows of fixed sizes), I've removed the application menu (`preferences > always show the window menu bar`) which apparently sets about a correct size for my windows to scroll correctly again.

Hence my best guess is that when your window size ends up having a last line overlapping the edge (may not be a regular message text line but some other elements far up causing it and pushing into that position, as I don't have *all* conversations always doing this, just all causing it at random times), the "scroll to bottom" feature will always get stuck some pixels behind and then causes the problem described in the comments above.

**Comment by progserega** [comment]
In my case settings "always show menu" was on. And with this settings - in some rooms button-circle "go down" show always and do not hide after press on it. If I disable options "always show menu" - this button hide in this "break room". If I press "alt" on keyboard for show menu - button show again.

I think this is bug in code, which calculate size of window and it is not use information about size of enabled menu.

Element desktop: 1.7.13
OS: Ubuntu GNU\Linux 16.0.4

**Comment by otheus** [comment]
"me too" / again? I felt this problem was fixed in a prior release, but it seems to have re-appeared in 1.7.34, "olm version 3.2.3". MacOS Catalina.

**Comment by dkoch84** [comment]
I'm seeing this too. Element Desktop 1.7.34. In my case, it appears to be caused by enabling "Jump to the bottom of the timeline when you send a message" All Settings > Preferences > Timeline

Perhaps others can verify.

**Comment by kittykat** [comment]
Hi all, if anyone has time to figure out exact and consistent reproduction steps for this issue on develop.element.io and document them here, that would help progress this ticket. We are looking for [this level of detail](https://github.com/vector-im/element-web/issues/19137#issuecomment-926170071) in the steps.
